docs/build-and-publish-pipeline.md

```diff
@@ -73,7 +73,7 @@ For iOS consumers the corresponding steps are:
     * TODO: could a malicious dev dependency from step (3) influence the build environment here?
 5. CircleCI uses [dpl](https://github.com/travis-ci/dpl) to publish to GitHub as a release artifact.
     * See [Authentication and secrets below](#authentication-and-secrets)
-6. Consumers add Application services as a dependency from the [Rust Components Swift](https://github.com/mozilla/rust-components-swift/) repo using Apple's Swift Package Manager.
+6. Consumers can add Application services as a dependency in their corresponding swift package. See how [firefox-ios](https://github.com/mozilla-mobile/firefox-ios/tree/main/MozillaRustComponents) does it for more information. 
```

Comment:
nit (maybe): Take this with a grain of salt because what I'm about to say applies both to this update and the original wording: This step abstracts away so much detail that I'm not sure it's helpful.

But since there probably aren't going to be a lot of new iOS consumers of A-S anytime soon maybe something like

> Consumers can add the Application Services rust components as a dependency in their corresponding swift package by replicating the firefox-ios integration [here](https://github.com/mozilla-mobile/firefox-ios/tree/main/MozillaRustComponents) .

I'm just trying to clarify what `MozillaRustComponents` is. If I didn't know what I know I might be stumped at how Application services connects to `MozillaRustComponents`. IDK, grain of salt, etc.